File: python-practice/C vs PY/million.c
#include <stdio.h>
#include <stdbool.h>
#include <time.h>

int main(void)
{
    clock_t start, end;
    double timeTaken;
    start = clock();
    long unsigned i = 1;

    while (true)
    {
        printf("%ld\n", i);
        i += 1;
        if (i == 1000001)
        {
            break;
        }
    }
    end = clock();
    timeTaken = ((double)(end - start)) / CLOCKS_PER_SEC;
    printf("Time taken: %f seconds\n", timeTaken);
    return (0);
}

File: python-practice/C vs PY/million.py
#!/usr/bin/python3
import time as t


def main():
    start = t.time()
    i = 1

    while True:
        print(i)
        i += 1
        if i == 1000001:
            break

    end = t.time()
    print("Time taken: {} seconds".format(end - start))


main()
